examples/suit_update/main.c

```diff
@@ -48,6 +53,15 @@ static msg_t _nanocoap_server_msg_queue[NANOCOAP_SERVER_QUEUE_SIZE];
 #define MAIN_QUEUE_SIZE     (8)
 static msg_t _main_msg_queue[MAIN_QUEUE_SIZE];
 
+/* add handled storages */
+#if IS_USED(MODULE_SUIT_STORAGE_VFS)
+XFA_USE(char*, suit_storage_files_reg);
+#ifdef BOARD_NATIVE
```

Comment:
Why limit this to `native`?

Reply:
I'm just adding this storage for native in the test, same question can be asked for RAM storage